openid-sharedsignals-framework-1_0.md

```diff
@@ -762,6 +764,12 @@ which can be used by Event Receivers to create and delete one or more Event Stre
 The API can also be used to query and update the Event Stream's configuration and status,
 add and remove Subjects, and trigger verification for those streams.
 
+Unless there exists some other method of establishing trust between a Transmitter and
+Receiver, all Stream Management API endpoints MUST use HTTP over TLS {{RFC2818}}
+and standard HTTP authentication and authorization schemes, as per {{RFC7235}}.
+The authorization MUST associate a Receiver with one or more stream IDs, such that only
```

Comment:
"The authorization" reads a bit weird to me but not sure I have a better suggestion.

Reply:
Would you prefer "This authorization"?